src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -2019,9 +2019,27 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
         }
         const result = await this.query(`SELECT "udt_schema", "udt_name" ` +
             `FROM "information_schema"."columns" WHERE "table_schema" = '${schema}' AND "table_name" = '${name}' AND "column_name"='${column.name}'`);
+
+        let udtName = result[0]["udt_name"];
+
+        // You can not modify the array type. You need to edit the enum itself.
+        if (column.isArray) {
+            // Find the information about the array type
+            const typeResult = await this.query(`SELECT "typelem" FROM pg_type WHERE "typname" = $1`, [udtName]);
+
+            const typelem = typeResult[0]["typelem"];
+            // Get with the oid of the enum array type the enum type
+            const baseTypeNameResult = await this.query(`SELECT "typname" FROM pg_type WHERE oid = $1`, [typelem]);
+
+            const baseTypeName = baseTypeNameResult[0]["typname"];
```

Comment:
Is there a case where there won't be at least 1 value in this result set?

Reply:
Should always be 1 or zero.

Reply:
If there are 0 went this fail with an index error?  Is that what we want?

Reply:
I mean, it can only fail if there is no array enum. It should always be 1 if there is one. And if not, then there is already something wrong somewhere else. Should I throw an actual error or what is your suggestion?

Reply:
Could there be no array enum if you are making a column an array?

Reply:
I don't think so.